model-engine/model_engine_server/infra/services/image_cache_service.py

```diff
@@ -92,7 +92,7 @@ def _cache_finetune_llm_images(
             latest_tag,
         )
         forwarder_image = DockerImage(
-            f"{infra_config().docker_repo_prefix}/launch/gateway", GIT_TAG
+            f"{infra_config().docker_repo_prefix}/model-engine/gateway", GIT_TAG
```

Comment:
should it be `model-engine/gateway` or `model-engine`? I don't see a repository in ECR called `model-engine/gateway`. Only `model-engine` or `image-builder/gateway`, `launch/gateway`, and `train/gateway`

Reply:
good catch. it should be model-engine